apps/namada-interface/src/App/AccountOverview/DerivedAccounts/DerivedAccounts.components.ts

```diff
@@ -1,19 +1,26 @@
 import styled from "styled-components/macro";
-import { DesignConfiguration } from "utils/theme";
+import { ColorMode, DesignConfiguration } from "utils/theme";
 
 enum ComponentColor {
   ScrollbarThumb,
 }
 
 const getColor = (
-  color: ComponentColor,
+  toggleColor: ComponentColor,
```

Comment:
nit: I feel that the naming here might be `forComponent`, but the old `color` from me was also too ambiguous.